### DISCUSSION TITLE
Custom FBW ec5f585

### DISCUSSION THREAD

**Discussion by luiskjr** [flybywiresim/aircraft#3844] [Help] [answered]
I wanted to use the version with the new electric update but i get a NO cockpit when launching

The stable version works fine

Used the setup.exe included

Any ideas why?

**Comment by ChruutvoLuzi** [comment, marked as answer]
Do you have the copilot mod installed? I remember that being an issue

**Reply by luiskjr** [reply]
yes i have, thanks i will try without it
